relay-event-normalization/src/event.rs

```diff
@@ -1,23 +1,13 @@
-//! Event normalization processor.
-//!
-//! This processor is work in progress. The intention is to have a single
-//! processor to deal with all event normalization. Currently, the normalization
-//! logic is split across several processing steps running at different times
-//! and under different conditions, like light normalization and store
-//! processing. Having a single processor will make things simpler.
-
```

Comment:
It's still would be great to keep some docs  describing what this module is responsible for.